packaging/installer/kickstart.sh

```diff
@@ -1423,6 +1450,12 @@ confirm_root_support
 get_system_info
 confirm_install_prefix
 
+if [ "${ACTION}" = "uninstall" ]; then
+  uninstall
+  trap - EXIT
+  exit 0
+fi
```

Comment:
There should be a call to `cleanup` here before exiting (should be before the `trap - EXIT` line).